ipatests/test_integration/test_acme.py

```diff
@@ -114,9 +105,9 @@ def prepare_acme_client(cls):
         cls.acme_server = f'https://{acme_host}/acme/directory'
 
         # install acme client packages
-        if not skip_certbot_tests:
-            tasks.install_packages(cls.clients[0], ['certbot'])
-        if not skip_mod_md_tests:
+        if osinfo.id == 'fedora':
+            tasks.install_packages(cls.clients[0], ['certbot', 'mod_md'])
+        if osinfo.id == 'rhel':
```

Comment:
By dropping the global variables this is going to break the test on non-fedora and RHEL platforms because they will no longer be marked as skip.

I think I'd leave it as it was and add a special case here for rhel since it the package will be provided externally via pip.